src/lib_c/x86_64-windows-msvc/c/sysinfoapi.cr

```diff
@@ -1,9 +1,6 @@
 require "c/winbase"
 
 lib LibC
-  fun GetSystemTimeAsFileTime(time : FILETIME*)
-  fun GetSystemTimePreciseAsFileTime(time : FILETIME*)
```

Comment:
These functions are both defined in `sysinfoapi.h`, so it looks like they should stay here and the duplicates in `winbase.cr` be removed.